tensorboard/pip_package/README.rst

```diff
@@ -1,2 +1,5 @@
 TensorBoard is a suite of web applications for inspecting and understanding
 your TensorFlow runs and graphs.
+
+Releases prior to 1.6.0 were published under the ``tensorflow-tensorboard`` name
```

Comment:
nit: I think 1 back tick suffices.

Reply:
Nope, this is reStructuredText and you need two backticks for monospaced literals:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup

Reply:
Ah, thanks!